core/src/main/java/com/linecorp/armeria/common/AbstractRequestContextBuilder.java

```diff
@@ -433,6 +435,24 @@ protected final String path() {
         return path;
     }
 
+    /**
+     * Sets the {@link UUID} of the request. If not set, it is auto-generated with {@link UUID}
```

Comment:
it is auto-generated with ... -> `a random {@link UUID} is generated with {@link UUID#randomUUID()}.`